testing/environments/alpha2.yml

```diff
@@ -4,7 +4,7 @@
 elasticsearch:
   build: ./docker/elasticsearch
   dockerfile: Dockerfile-5.0.0-alpha1    # still using alpha1 because alpha2 doesn't work fine in docker
-  command: elasticsearch -Ees.network.host=0.0.0.0 -Ees.discovery.zen.minimum_master_nodes=1
+  command: elasticsearch -Enetwork.host=0.0.0.0 -Ediscovery.zen.minimum_master_nodes=1
```

Comment:
@tsg Agree, we should just remove alpha1 and alpha2 as we don't need it anymore. I can do it in this PR here: https://github.com/elastic/beats/pull/1724